docs/email-signup.md

```diff
@@ -0,0 +1,23 @@
+# Email signup
+
+Currently only the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) is making use of these features, but other finders could use these in the future.
+
+## Subscribing to multiple facet values
+
+Users can [pick and choose which `facet_values` to subscribe to](https://github.com/alphagov/finder-frontend/pull/1191), and will be emailed according to their chosen frequency whenever content is newly tagged or existing tagged content is updated (provided the 'send email' checkbox is checked in content tagger).
+
+The ability to choose facet values from across multiple facets was added in [email-alert-api#867](https://github.com/alphagov/email-alert-api/pull/867) and simplified in [email-alert-api#892](https://github.com/alphagov/email-alert-api/pull/892). This is how the email alert subscription logic works:
+
+- A piece of content is updated, triggering the [notification handler service](https://github.com/alphagov/email-alert-api/blob/188cd1c3c855976e276f8a90714ca4e6c5e0d825/app/services/notification_handler_service.rb).
+- email-alert-api [fetches the links and tags associated with the content](https://github.com/alphagov/email-alert-api/blob/5cc5c23d0c6de80aaa64dbb28722ca705a78732d/app/services/matched_content_change_generation_service.rb#L32-L38) (amongst other things).
+- The `SubscriberList` table is searched for any subscriber lists that is [associated with at least one of the links](https://github.com/alphagov/email-alert-api/blob/5cc5c23d0c6de80aaa64dbb28722ca705a78732d/app/queries/subscriber_list_query.rb#L25), or with at least one of the tags, or with the document type.
+- Any [duplicate lists are filtered out](https://github.com/alphagov/email-alert-api/blob/5cc5c23d0c6de80aaa64dbb28722ca705a78732d/app/queries/subscriber_list_query.rb#L11-L15).
+- The [email is scheduled for sending out](https://github.com/alphagov/email-alert-api/blob/188cd1c3c855976e276f8a90714ca4e6c5e0d825/app/services/notification_handler_service.rb#L15).
+
+For example, if a user has subscribed to updates about `Aerospace` or `EU Citizens`, and a piece of content linked to `EU Citizens` is updated, the 'at least one link is associated with the content' condition passes and an email will be sent to the user.
+
+Now that subscriptions can be custom selected by the user, rather than curated by a content editor, the subscription title could become very long and difficult to read if selecting too many facet values. Therefore, there is now an [`EmailAlertListTitleBuilder` for creating comma-separated titles](https://github.com/alphagov/finder-frontend/pull/1164) that are easier to read. In the spirit of optimising code for deletion, much of the code is in separate files to the main title logic.
```

Comment:
I don't think you need this as the commit message has the same information: https://github.com/alphagov/finder-frontend/commit/f538171b0a90a63674e8d6869ce17b2dfaa69099